README.md

```diff
@@ -20,11 +19,18 @@ Zonemaster LDNS
 
 ## Introduction
 
-This module provides a Perl interface to the [ldns library](https://www.nlnetlabs.nl/projects/ldns/) from [NLnet Labs](https://www.nlnetlabs.nl/). This module includes the necessary C code from ldns, so the library does not need to be globally installed. It does, however, rely on a sufficiently recent version of OpenSSL being present.
+This module provides a Perl interface to the [ldns library] from [NLnet Labs].
+This module includes the necessary C code from ldns, so the library does not need
+to be globally installed. It does, however, rely on a sufficiently recent version
+of OpenSSL being present.
 
-This module is written as part of the [Zonemaster project](http://github.com/zonemaster/zonemaster), and therefore primarily exposes the functionality needed for that. Since Zonemaster is a diagnostic tool, that means the functions most used are those for looking things up and inspecting them.
+This module is written as part of the [Zonemaster project], and therefore
+primarily exposes the functionality needed for that. Since Zonemaster is a
+diagnostic tool, that means the functions most used are those for looking things
+up and inspecting them.
```

Comment:
We've been using vanilla ldns for years, haven't we? I think we could remove this section.

Reply:
Yes, but that is not how I read it. This Perl integration of LDNS is not generic. It has been created to meet the needs of Zonemaster. That is still true, isn't it?

Reply:
Yes, I'm sure it was created because Zonemaster had needs that ldns didn't provide for. When you and I came into the Zonemaster project we still had a bunch of our own ldns patches. That's still true. But since then the ldns guys have integrated all the changes we needed in the C code.

Do you know if we have something in the XS code that's lacking in DNS::LDNS? I've actually never compared them. The only thing I'm sure isn't generic about Zonemaster::LDNS is the purpose.

Reply:
No, I do not think there is anything additional in the XS code. It is rather what is exposed and how compared to DNS::LDNS.

Reply:
I guess the text is fine as is for now?